specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -0,0 +1,1290 @@
+        }
+      },
+      "description": "Describes the properties of a Compute Operation value."
+    },
+    "ComputeOperationValueDisplay": {
+      "properties": {
+        "operation": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The display name of the compute operation."
+        },
+        "resource": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The display name of the resource the operation applies to."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The description of the operation."
+        },
+        "provider": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource provider for the operation."
+        }
+      },
+      "description": "Describes the properties of a Compute Operation Value Display."
+    },
+    "Resource": {
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "ContainerServiceOSDisk": {
+      "type": "integer",
+      "format": "int32",
+      "maximum": 1023,
+      "minimum": 0,
+      "description": "OS Disk Size in GB to be used to specify the disk size for every machine in this master/agent pool. If you specify 0, it will apply the default osDisk size according to the vmSize specified."
+    },
+    "ContainerServiceStorageProfile": {
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ContainerServiceStorageProfileTypes",
+        "modelAsString": true
+      },
+      "enum": [
+        "StorageAccount",
+        "ManagedDisks"
+      ],
+      "description": "Storage profile specifies what kind of storage used. Choose from StorageAccount and ManagedDisks. Leave it empty, we will choose for you based on the orchestrator choice."
+    },
+    "ContainerServiceVnetSubnetID": {
+      "type": "string",
+      "description": "VNet SubnetID specifies the vnet's subnet identifier. If you specify either master VNet Subnet, or agent VNet Subnet, you need to specify both. And they have to be in the same VNet."
```

Comment:
Please remove "If you specify either master VNet Subnet, or agent VNet Subnet, you need to specify both. And they have to be in the same VNet.". In AKS case, we only need the agent subnet ID.